app/controllers/users.js

```diff
@@ -0,0 +1,13 @@
+const usersService = require('../services/users');
+
+const signUp = async ({ body }, res) => {
+  try {
+    const user = await usersService.createUser(body);
+    console.info(`username: ${user.email}`);
+    res.status(201).send(user);
+  } catch (e) {
+    res.status(400).send(e.message);
```

Comment:
Again, this is fine, but with express middlewares you can add error handling, this base project has it (`app/middlewares/errors.js`). To use it in any controller you would have to make use of the `next()` express function, this function detects if what you send to it is an error and routes it to the error handling middleware. Do that here :D